src/components/flyout/Flyout.scss

```diff
@@ -22,35 +24,47 @@
             @include bdl-openComponentAnimation($bdl-transitionDurationBase);
         }
     }
-
-    .bdl-OverlayHeader {
-        display: none;
-    }
 }
 
-@include breakpoint($medium-screen) {
-    .bdl-Flyout--responsive {
-        .flyout-overlay-enabled {
-            // cancels out the tether inline styling without having to set enabled=false
-            transform: none !important;
-        }
+body.is-responsive-web {
```

Comment:
I think this is a safeguard to prevent the CSS from applying if is-responsive-web is declared somewhere else. This is what the DejaBlue team did I believe

Reply:
On a second thought, I will handle this in EUA. I feel it is better practice and easier to clean